src/libcore/scanAccountsOnDevice.js

```diff
@@ -93,7 +94,7 @@ export const scanAccountsOnDevice = (
         o.complete();
       } catch (e) {
         const mappedError = remapLibcoreErrors(e);
-
+        Sentry.captureException(mappedError);
```

Comment:
prefer passing the `e` that is the raw error

Reply:
On it!